src/routes/member/[studentId]/+page.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
```

Comment:
These pages do get a bit long. I think we should consider composing components, either by just splitting stuff up into components, or additionally using slots.